### DISCUSSION TITLE
Telegram PrivateGroup

### DISCUSSION THREAD

**Discussion by TurkGSMdev** [FranzKafkaYu/x-ui#198] [Show and tell]
Hello,
I was not approved by Telegram bot to enter your private group
FranzKafka'sPrivateGroup
plz plz plz help me
![image_2022-12-04_13-47-40](https://user-images.githubusercontent.com/119725375/205485654-f6754e99-34d1-4936-984f-ff8455a1ba8f.png)

my Users in TG:
https://t.me/m_ali_turkgsm
https://t.me/xiaomi_store09367155070

**Comment by FranzKafkaYu** [comment]
@TurkGSMdev  you need to apply to join group.I didn't find you in this group now.

**Reply by TurkGSMdev** [reply]
my name:
TurkGSM admin
UserName:
@m_ali_turkgsm


![photo_2022-12-04_19-08-28](https://user-images.githubusercontent.com/119725375/205500097-f8bdc9fd-1f8e-4891-8b95-f9d368a3f18d.jpg)
